tool/IRsend_test.h

```diff
@@ -0,0 +1,104 @@
+// Copyright 2017 David Conran
+
+#ifndef TEST_IRSEND_TEST_H_
```

Comment:
Can you reference the canonical file instead of making essentially a copy of it here? It would be better if you just included the "real" IRsend_test.h file instead of making a copy which we need to keep in sync etc.